tests/builtins/test_ascii.py

```diff
@@ -2,28 +2,45 @@
 
 
 class AsciiTests(TranspileTestCase):
-    pass
+    def test_ascii_without_unicode(self):
+        self.assertCodeExecution("""
+            print(ascii("Hello World!"))
+            print(ascii('\u0041'))
+
+            string = ''
+            for i in range(100):
```

Comment:
It looks like going to 100 won't be sufficient to fully exercise your implementation of `ascii()` – it seems to have different logic for 32, 127, 256 and more?